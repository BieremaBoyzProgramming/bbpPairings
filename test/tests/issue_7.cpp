// https://github.com/BieremaBoyzProgramming/bbpPairings/issues/7
void TEST_FUNCTION()
{
  auto output_path = testing::data_folder_path + STRINGIFY(TEST_ID) ".output";
  testing::run(
    testing::exe_path
    + " --dutch "
    + testing::data_folder_path
    + STRINGIFY(TEST_ID) ".input -p "
    + output_path);
  testing::assert_file_content_matches
    (output_path, testing::data_folder_path + STRINGIFY(TEST_ID) ".output.expected");
}
