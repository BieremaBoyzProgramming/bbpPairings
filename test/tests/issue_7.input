012 AutoTest Tournament 1110065304001    1      Test0001 Player0001               2720                            10.5    1    31 b 1    16 w 1    12 b 1     6 w 1     7 b 0     4 w 1     3 b 0    14 w 1    10 w 1     8 b 1     5 w 1    11 b 0    17 w =     2 b 1001    2      Test0002 Player0002               2701                             8.5   11    32 w 1    17 b 0    29 w 1    15 b =    27 w 1    13 b 1     8 w =    11 b 0    20 w 1     7 b =    10 w =    12 b =     9 w 1     1 w 0001    3      Test0003 Player0003               2697                            10.0    2    33 b 1    18 w =    22 b 1    26 w 1     8 b 1     7 w =     1 w 1     6 b 1    11 w 1     5 b 0    12 w 0    17 b =     4 b 1    10 w =001    4      Test0004 Player0004               2689                             9.0    6    34 w +    19 b 1    11 w =    17 b =     5 w 1     1 b 0    18 w =     8 b -    27 b 1    15 w +     7 w 1     8 b =     3 w 0    14 b 1001    5      Test0005 Player0005               2673                             9.0    7    35 b 1    20 w 1    14 b 1     8 w =     4 b 0     6 w -    17 w 1    18 b 1    16 w +     3 w 1     1 b 0     7 w =    10 w 1     6 b 0001    6      Test0006 Player0006               2664                             9.0    8    36 w 1    21 b 1    13 w 1     1 b 0    12 w 1     5 b +     7 b 0     3 w 0    31 b 1    11 b =     8 w 0    15 b 1    16 w =     5 w 1001    7      Test0007 Player0007               2655                             9.5    3    37 b 1    22 w =    30 b +    11 b 1     1 w 1     3 b =     6 w 1    10 w +     8 b 0     2 w =     4 b 0     5 b =    12 w 1    17 b =001    8      Test0008 Player0008               2653                             9.5    4    38 w 1    25 b 1    15 w 1     5 b =     3 w 0    17 b 1     2 b =     4 w +     7 w 1     1 w 0     6 b 1     4 w =    11 b 1    12 b 0001    9      Test0009 Player0009               2636                             8.0   13    39 b -    53 w 1    31 b 1    19 b 1    17 w =    14 w =    10 b 0    25 w 1    22 b 1    12 b =    11 w 0    13 w 1     2 b 0    15 b =001   10      Test0010 Player0010               2596                             8.5   12    40 w 1    26 b =    17 w =    21 b 1    13 w =    15 b 1     9 w 1     7 b -     1 b 0    16 w 1     2 b =    20 w 1     5 b 0     3 b =001   11      Test0011 Player0011               2593                             9.5    5    41 b 1    24 w 1     4 b =     7 w 0    14 b =    29 w 1    20 b 1     2 w 1     3 b 0     6 w =     9 b 1     1 w 1     8 w 0    16 b 1001   12      Test0012 Player0012               2583                             9.0    9    42 w 1    27 b 1     1 w 0    23 b 1     6 b 0    18 w =    31 b =    26 w 1    14 b 1     9 w =     3 b 1     2 w =     7 b 0     8 w 1001   13      Test0013 Player0013               2569                             8.0   14    43 b 1    28 w 1     6 b 0    16 w 1    10 b =     2 w 0    27 b 1    20 w -    29 w 1    17 b 0    39 w 1     9 b 0    22 w =    36 b 1001   14      Test0014 Player0014               2568                             8.0   15    44 w 1    29 b 1     5 w 0    25 b 1    11 w =     9 b =    21 w 1     1 b 0    12 w 0    26 b 1    17 w 0    27 b +    20 b 1     4 w 0001   15      Test0015 Player0015               2548                             8.0   16    45 b 1    39 w 1     8 b 0     2 w =    26 b 1    10 w 0    33 b 1    16 w -    18 w 1     4 b -    25 b 1     6 w 0    31 b 1     9 w =001   16      Test0016 Player0016               2541                             8.0   17    46 w 1     1 b 0    38 w 1    13 b 0    40 w =    30 b 1    23 w 1    15 b +     5 b -    10 b 0    31 w 1    19 b 1     6 b =    11 w 0001   17      Test0017 Player0017               2521                             9.0   10    47 b 1     2 w 1    10 b =     4 w =     9 b =     8 w 0     5 b 0    39 w 1    33 b 1    13 w 1    14 b 1     3 w =     1 b =     7 w =001   18      Test0018 Player0018               2419                             8.0   18    48 w 1     3 b =    26 w 0    33 b 1    42 w 1    12 b =     4 b =     5 w 0    15 b 0    39 w -    36 w =    43 b 1    39 w 1    21 b 1001   19      Test0019 Player0019               2386                             7.0   25    49 b 1     4 w 0    35 b 1     9 w 0    31 b 0    39 w =    43 b -    32 b 1    42 w 1    40 b 1    21 w =    16 w 0    34 b 1    25 w -001   20      Test0020 Player0020               2342                             7.5   22    50 w 1     5 b 0    40 w =    36 b =    35 w 1    25 b 1    11 w 0    13 b +     2 b 0    28 w 1    22 b 1    10 b 0    14 w 0    29 b =001   21      Test0021 Player0021               2326                             7.0   26    51 b 1     6 w 0    37 b 1    10 w 0    39 b 1    24 w 1    14 b 0    31 w 0    34 b 1    25 w =    19 b =    22 w 0    32 b 1    18 w 0001   22      Test0022 Player0022               2317                             8.0   19    52 w 1     7 b =     3 w 0    40 b =    36 w =    38 b =    32 w 1    30 b 1     9 w 0    29 b 1    20 w 0    21 b 1    13 b =    23 w =001   23      Test0023 Player0023               2296                             7.5   23    53 b =    30 w =    39 b 1    12 w 0    37 b 1    31 w =    16 b 0    33 w 0    35 b =    41 w 1    29 w =    38 b 1    26 w =    22 b =001   24      Test0024 Player0024               2286                             8.0   20    54 w 1    11 b 0    42 w =    44 b 1    30 w =    21 b 0    40 w 1    29 b 0    36 w 1    31 b 0    38 w =    39 b =    35 w 1    26 b 1001   25      Test0025 Player0025               2272                             8.0   21    55 b 1     8 w 0    41 b 1    14 w 0    46 b 1    20 w 0    34 w 1     9 b 0    30 w 1    21 b =    15 w 0    40 b 1    29 w =    19 b +001   26      Test0026 Player0026               2257                             7.0   27    56 w +    10 w =    18 b 1     3 b 0    15 w 0    36 b =    38 w 1    12 b 0    43 w 1    14 w 0    33 b 1    31 w =    23 b =    24 w 0001   27      Test0027 Player0027               2248                             6.0   40    57 b +    12 w 0    43 b =    47 w 1     2 b 0    42 b 1    13 w 0    40 b 1     4 w 0    33 w =    28 b =    14 w -    36 w 0    45 b =001   28      Test0028 Player0028               2229                             7.0   28    58 w 1    13 b 0    44 w =    42 b 0    60 w 1    40 b 0    48 w 1    43 b =    38 w 1    20 b 0    27 w =    29 b 0    53 w =    46 b 1001   29      Test0029 Player0029               2212                             7.5   24    59 b +    14 w 0     2 b 0    45 w 1    41 b 1    11 b 0    36 w 1    24 w 1    13 b 0    22 w 0    23 b =    28 w 1    25 b =    20 w =001   30      Test0030 Player0030               2199                             6.0   41    60 w 1    23 b =     7 w -    43 w 1    24 b =    16 w 0    46 b 1    22 w 0    25 b 0    47 w =    51 b =    35 w -    48 w 1    39 b 0001   31      Test0031 Player0031               2198                             7.0   29     1 w 0    46 b 1     9 w 0    48 b 1    19 w 1    23 b =    12 w =    21 b 1     6 w 0    24 w 1    16 b 0    26 b =    15 w 0    38 b =001   32      Test0032 Player0032               2189                             6.5   35     2 b 0    47 w =    49 b =    46 w 0    53 b 1    41 w 1    22 b 0    19 w 0    55 b +    38 b 0    48 w 1    44 b 1    21 w 0    37 w =001   33      Test0033 Player0033               2185                             7.0   30     3 w 0    48 b =    60 w 1    18 w 0    49 b 1    44 b 1    15 w 0    23 b 1    17 w 0    27 b =    26 w 0    36 b 0    55 w 1    53 b +001   34      Test0034 Player0034               2170                             6.0   42     4 b -    49 w =    47 b 0    53 w =    54 b 1    45 w 1    25 b 0    42 b 1    21 w 0    43 b =    44 w =    51 b 1    19 w 0    40 b 0001   35      Test0035 Player0035               2163                             6.5   36     5 w 0    50 b 1    19 w 0    52 b 1    20 b 0    46 w 0    47 b -    57 b 1    23 w =    60 b =    58 w 1    30 b +    24 b 0    44 w =001   36      Test0036 Player0036               2155                             7.0   31     6 b 0    51 w =    53 b 1    20 w =    22 b =    26 w =    29 b 0    47 w 1    24 b 0    37 w =    18 b =    33 w 1    27 b 1    13 w 0001   37      Test0037 Player0037               2141                             6.5   37     7 w 0    52 b 1    21 w 0    54 b 1    23 w 0    48 b 0    53 w =    58 b =    50 w 1    36 b =    43 w 0    42 b 1    40 w =    32 b =001   38      Test0038 Player0038               2118                             7.0   32     8 b 0    55 w 1    16 b 0    49 w =    47 b 1    22 w =    26 b 0    46 w 1    28 b 0    32 w 1    24 b =    23 w 0    45 b +    31 w =001   39      Test0039 Player0039               2104                             7.0   33     9 w +    15 b 0    23 w 0    56 b 1    21 w 0    19 b =    44 w 1    17 b 0    60 w 1    18 b +    13 b 0    24 w =    18 b 0    30 w 1001   40      Test0040 Player0040               2074                             7.0   34    10 b 0    56 w 1    20 b =    22 w =    16 b =    28 w 1    24 b 0    27 w 0    41 b 1    19 w 0    47 b 1    25 w 0    37 b =    34 w 1001   41      Test0041 Player0041               2065                             6.0   43    11 w 0    54 b 1    25 w 0    58 b 1    29 w 0    32 b 0    57 w 1    48 b 1    40 w 0    23 b 0    42 w =    53 b 0    51 w =    43 b 1001   42      Test0042 Player0042               2064                             6.0   44    12 b 0    57 w 1    24 b =    28 w 1    18 b 0    27 w 0    60 b 1    34 w 0    19 b 0    45 w =    41 b =    37 w 0    50 b 1    51 w =001   43      Test0043 Player0043               2049                             5.5   51    13 w 0    58 b 1    27 w =    30 b 0    44 w 0    52 b 1    19 w +    28 w =    26 b 0    34 w =    37 b 1    18 w 0    46 b 0    41 w 0001   44      Test0044 Player0044               2040                             6.5   38    14 b 0    59 w 1    28 b =    24 w 0    43 b 1    33 w 0    39 b 0    55 w =    45 b =    54 w 1    34 b =    32 w 0    47 w 1    35 b =001   45      Test0045 Player0045               2035                             6.0   45    15 w 0    60 b =    48 w =    29 b 0    56 w 1    34 b 0    58 w =    53 b =    44 w =    42 b =    55 w 1    47 b =    38 w -    27 w =001   46      Test0046 Player0046               2026                             6.5   39    16 b 0    31 w 0    51 b +    32 b 1    25 w 0    35 b 1    30 w 0    38 b 0    48 w 0    53 b =    60 w 1    59 b 1    43 w 1    28 w 0001   47      Test0047 Player0047               2025                             6.0   46    17 w 0    32 b =    34 w 1    27 b 0    38 w 0    57 b =    35 w +    36 b 0    53 w 1    30 b =    40 w 0    45 w =    44 b 0    56 b 1001   48      Test0048 Player0048               2004                             6.0   47    18 b 0    33 w =    45 b =    31 w 0    55 b 1    37 w 1    28 b 0    41 w 0    46 b 1    51 w 0    32 b 0    60 w 1    30 b 0    57 w 1001   49      Test0049 Player0049               1945                             5.0   52    19 w 0    34 b =    32 w =    38 b =    33 w 0    60 b 0    55 w 0    50 b 0    56 w =    57 b =    59 w 0    52 b 1    54 w 1    58 b =001   50      Test0050 Player0050               1939                             5.0   53    20 b 0    35 w 0    56 b -    57 b =    59 w 0    51 b =    56 w 1    49 w 1    37 b 0    55 b 0    52 w =    58 b =    42 w 0    54 w 1001   51      Test0051 Player0051               1925                             6.0   48    21 w 0    36 b =    46 w -    60 b 0    57 w 0    50 w =    59 b 1    54 w =    52 b 1    48 b 1    30 w =    34 w 0    41 b =    42 b =001   52      Test0052 Player0052               1916                             4.5   55    22 b 0    37 w 0    57 b 1    35 w 0    58 b =    43 w 0    54 b =    60 w -    51 w 0    56 b 1    50 b =    49 w 0    59 w 1    55 b 0001   53      Test0053 Player0053               1909                             6.0   49    23 w =     9 b 0    36 w 0    34 b =    32 w 0    59 b 1    37 b =    45 w =    47 b 0    46 w =    54 b 1    41 w 1    28 b =    33 w -001   54      Test0054 Player0054               1896                             4.0   59    24 b 0    41 w 0    59 b 1    37 w 0    34 w 0    55 b =    52 w =    51 b =    58 w 1    44 b 0    53 w 0    56 w =    49 b 0    50 b 0001   55      Test0055 Player0055               1836                             6.0   50    25 w 0    38 b 0    58 w 0    59 b 1    48 w 0    54 w =    49 b 1    44 b =    32 w -    50 w 1    45 b 0    57 w 1    33 b 0    52 w 1001   56      Test0056 Player0056               1824                             3.5   60    26 b -    40 b 0    50 w +    39 w 0    45 b 0    58 w =    50 b 0    59 w 0    49 b =    52 w 0    57 w 0    54 b =    60 b 1    47 w 0001   57      Test0057 Player0057               1801                             4.5   56    27 w -    42 b 0    52 w 0    50 w =    51 b 1    47 w =    41 b 0    35 w 0    59 b 0    49 w =    56 b 1    55 b 0    58 w 1    48 b 0001   58      Test0058 Player0058               1797                             5.0   54    28 b 0    43 w 0    55 b 1    41 w 0    52 w =    56 b =    45 b =    37 w =    54 b 0    59 w 1    35 b 0    50 w =    57 b 0    49 w =001   59      Test0059 Player0059               1792                             4.5   57    29 w -    44 b 0    54 w 0    55 w 0    50 b 1    53 w 0    51 w 0    56 b 1    57 w 1    58 b 0    49 b 1    46 w 0    52 b 0    60 w =001   60      Test0060 Player0060               1787                             4.5   58    30 b 0    45 w =    33 b 0    51 w 1    28 b 0    49 w 1    42 w 0    52 b +    39 b 0    35 w =    46 b 0    48 b 0    56 w 0    59 b =XXR 15