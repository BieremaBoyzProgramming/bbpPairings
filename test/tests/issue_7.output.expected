30
1 15
3 2
11 17
7 10
8 14
4 6
5 12
9 16
13 25
24 22
18 29
20 23
19 33
21 38
39 26
28 36
31 40
37 35
44 46
30 32
53 42
47 48
51 55
27 50
34 58
49 45
41 59
56 43
60 52
54 57
